packages/openai-compatible/src/openai-compatible-chat-language-model.ts

```diff
@@ -375,7 +376,16 @@ export class OpenAICompatibleChatLanguageModel implements LanguageModelV1 {
 
     const { args, warnings } = this.getArgs({ ...options });
 
-    const body = JSON.stringify({ ...args, stream: true });
+    const body = {
+      ...args,
+      stream: true,
+
+      // only include stream_options when in strict compatibility mode:
+      stream_options: this.config.includeUsage
+        ? { include_usage: true }
+        : undefined,
+    };
+
```

Comment:
might be good to have a test case for this since it might break w/ porting and refactorings

Reply:
good idea 🙏